readme.md

```diff
@@ -52,6 +52,13 @@ Default: `false`
 
 By default the wrap is soft, meaning long words may extend past the column width. Setting this to `true` will make it hard wrap at the column width.
 
+##### wordWrap
+
+Type: `boolean`<br>
+Default: `true`
+
+By default, a line will be split at a space. When set to `false`, it can break words into two as well.
```

Comment:
@sindresorhus as of right now, I believe we split only spaces, not at all word boundaries; so this is accurate. Having said this, I might put something like:

> By default, an attempt is made to split words at spaces, ensuring that they don't extend past the configured columns. If wordWrap is `false`, each column will instead be completely filled splitting words as necessary.

 ^ @SamVerschueren maybe this needs some more noodling, but I think it better describes what you need?